cmd/esmapping-generator/generator/esmapping_generator.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package generator
+
+import (
+	"fmt"
+
+	"github.com/jaegertracing/jaeger/cmd/esmapping-generator/app"
+	"github.com/jaegertracing/jaeger/cmd/esmapping-generator/app/renderer"
+	"github.com/jaegertracing/jaeger/pkg/es"
+	"github.com/jaegertracing/jaeger/plugin/storage/es/mappings"
+)
+
+func GenerateMappings(options app.Options) string {
```

Comment:
```suggestion
func GenerateMappings(options app.Options) (string, error) {
```
return proper errors and log them from the caller